### DISCUSSION TITLE
Did anything change regarding Webpack since the wiki page was last updated?

### DISCUSSION THREAD

**Discussion by RuurdBijlsma** [RSATom/WebChimera.js#135] [Q&A] [answered]
I got webchimera working in electron 11.1.0 on a regular project, but when I try to add it to a project using Webpack (a vue-electron-builder project) I can't get the package to load. 
When using webpack I get an error that the 'node module parse failed'.
I followed the steps from the wiki (https://github.com/RSATom/WebChimera.js/wiki/WebChimera.js-with-Webpack) but I think that is a bit outdated since the loaders key isn't allowed in webpack for me. When I use the config specified by webpack (https://webpack.js.org/loaders/node-loader/) I get a different error `exports is not defined` in chunk-vendors.js, which is probably due to some package using require syntax where it's not supposed to.

So the question is, does anyone know how to configure webpack to allow me to load the WebChimera.js.node file?

**Comment by RuurdBijlsma** [comment, marked as answer]
I ended up solving this in a very weird way. I built my own .node file from the .cmd file from this repo, if I require that with node-loader before using wcjs-prebuilt it works

import wc from 'node-loader!./js/WrapChimera/WebChimera.js.x64.node'

 I don't use this `wc` anywhere, but without that line the prebuilt version doens't work for me no idea why.

**Comment by RSATom** [comment]
Yes, all wikis are very outdated (it was updated last time some years ago). On other side I never used Webpack with node addons myself, so don't know how to do it right way...

**Reply by RuurdBijlsma** [reply]
Thanks for the reply! I have it working with webpack now, but the event onframeready gives the renderer a uint8array without the attached properties width height voffset upffset. Do you happen know what could be causing this? I do get those properties when not using webpack

**Comment by RSATom** [comment]
Sorry, I don't know.
